content/docs/help.md

```diff
@@ -7,20 +7,20 @@ aliases:
 
 ### Want to use cloud.gov?
 
-Start with our [interest form](https://docs.google.com/forms/d/e/1FAIpQLSevZfuJ_4KE-MZlm9gttYfsXQp0PJL7OR6k6LbZ9XnFn-oA6g/viewform) to tell us about your organization and applications</a>. We’ll schedule a call where we can go over needs, options, and questions. You can also email [cloud-gov-inquiries@gsa.gov](mailto:cloud-gov-inquiries@gsa.gov), including to request a [free sandbox space]({{< relref "free-limited-sandbox.md" >}}) (available to anyone with a federal government email address).
+If you have a U.S. federal email address, you can get access to a [free sandbox space]({{< relref "free-limited-sandbox.md" >}}) and try cloud.gov right away. You can also email [cloud-gov-inquiries@gsa.gov](mailto:cloud-gov-inquiries@gsa.gov) with questions to help you understand if cloud.gov is right for you. If you're ready to use cloud.gov, use our [interest form](https://docs.google.com/forms/d/e/1FAIpQLSevZfuJ_4KE-MZlm9gttYfsXQp0PJL7OR6k6LbZ9XnFn-oA6g/viewform) to tell us about your organization and applications</a>. We’ll schedule a call where we can go over needs, options, and any remaining questions, then get the [federal Interagency Agreement (IAA/MOU) process]({{< relref "overview/pricing/how-to-purchase.md" >}}) underway. 
```

Comment:
I'd like to point people at the interest form as the first link here, since I _most_ want people to fill out the interest form! I'll make a followup PR with this.